roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -144,22 +148,34 @@ HrmpCloseChannelRequests: map HrmpChannelId => Option<()>;
 HrmpCloseChannelRequestsList: Vec<HrmpChannelId>;
 
 /// The HRMP watermark associated with each para.
+/// Invariant: every para should be onboarded.
```

Comment:
It would be nice to have a little more description here, or a link to the guide if appropriate. It's not obvious what onboarding a para entails.

Reply:
Oh, this is such a valuable catch!